language-server/modules/langserver-core/src/test/resources/completion/class_def/config/config24.json

```diff
@@ -204,12 +204,16 @@
       "insertTextFormat": "Snippet"
     },
     {
-      "label": "return",
+      "label": "return;",
       "kind": "Snippet",
       "detail": "Statement",
       "sortText": "O",
+<<<<<<< HEAD
```

Comment:
Shall we resolve these conflicts in the test files.

Reply:
Done